src/test/groovy/org/gradle/profiler/ProfilerIntegrationTest.groovy

```diff
@@ -1605,4 +1605,21 @@ buildTarget {
         logFile.find("> org.gradle.profiler.number = 1").size() == 4
         logFile.find("> org.gradle.profiler.number = 2").size() == 4
     }
+
+    def "overrides jvm args"() {
+        given:
+        def scenarios = file('performance.scenario')
+        scenarios.text = """
+help {
+    tasks = ["help"]
+    jvm-args = ["-Xmx1m"]
```

Comment:
The out of memory error is coming from this test - Teamcity scrapes the log, sees it and reports it as an error. I'd instead use the MemoryMxBean in the build script to print out the max memory in order to verify the setting.

Reply:
We could of course deactivate this check, but I think writing the test without relying on an error would be cleaner.